mockito-core/build.gradle.kts

```diff
@@ -200,3 +201,53 @@ fun mockitoExtensionConfigFile(project: Project, mockitoExtension: String) =
     file(project.layout.buildDirectory.dir("generated/resources/ext-config/test/mockito-extensions/$mockitoExtension"))
 //</editor-fold>
 
+//<editor-fold defaultstate="collapsed" desc="Generate version class file">
+// Manifest cannot be used because Android strips the manifest from the jar.
+
+val generateVersionClass by tasks.registering(GenerateVersionClass::class)
+
+sourceSets.main {
+    java {
+        srcDir(generateVersionClass)
+    }
+}
+
+tasks.withType<Checkstyle> {
+    exclude("org/mockito/internal/util/MockitoVersion.java")
```

Comment:
Ultra-nit: define constant value to use both here and in the task implementation